python/packages/aws-lambda-sdk/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## 0.1.11 (2023-03-30)
+### ⚠ BREAKING CHANGES
```

Comment:
@Mmarzex Breaking changes should have a minor bump (technically it should have been v0.2 release)